pycbc/io/hdf.py

```diff
@@ -857,11 +857,13 @@ def to_coinc_xml_object(self, file_name):
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
 
-        ifos = list(self.sngl_files.keys())
-        proc_id = ligolw_process.register_to_xmldoc(outdoc, 'pycbc',
-                     {}, instruments=ifos, comment='', version=pycbc_version.git_hash,
-                     cvs_repository='pycbc/'+pycbc_version.git_branch,
-                     cvs_entry_time=pycbc_version.date).process_id
+        ifos = sorted(self.sngl_files)
+        proc_table = create_process_table(
```

Comment:
has create_process_table been imported (I guess this will show up when tested)

Reply:
:picard:

Reply:
:thanks: (I like that this showed up properly in the gwastro pycbc-github channel)